packages/react/src/primitives/View/View.tsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { convertStylePropsToStyleObj, getNonStyleProps } from '../shared/utils';
 import { ViewProps } from '../types/index';
+import postcssJs from 'postcss-js';
+import autoprefixer from 'autoprefixer';
+
+const prefixer = postcssJs.sync([autoprefixer]);
```

Comment:
I feel like it makes more sense to put the logic inside a util function and then import that function to use here. What do you think?

Reply:
Or maybe could wrap it outside `convertStylePropsToStyleObj`